fixes #72: A script teanslates English to other languages

Linfye opened the pull request:
<!---
Thank you for your pull request! 🚀
-->

### Contributor checklist

<!-- Please replace the empty checkboxes [] below with checked ones [x] accordingly. -->

- [] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

<!--
Describe briefly what your pull request proposes to change. Especially if you have more than one commit, it is helpful to give a summary of what your contribution is trying to solve.

Also, please describe shortly how you tested that your change actually works.
-->
The script can run on Google Colab and that's where I code on. Because running the script will take too much time, the translated_words are only a part of all the words. But it shows the feasibility of the program.

Looking forward to your code review
### Related issue

<!--- Scribe-Data prefers that pull requests be related to already open issues. -->
<!--- If applicable, please link to the issue by replacing ISSUE_NUMBER with the appropriate number below. -->
<!--- Feel free to delete this section if this does not apply. -->

- #ISSUE_NUMBER
#72

andrewtavis commented:
Thanks for this, @Linfye! I'll get back to you with a review soon! Once this is merged you'd be welcome to work on the other languages :)

wkyoshida reviewed (commented):
Awesome contribution @Linfye :rocket: Thanks so much for the work here!!

I'll let @andrewtavis add his review here as well, but just adding some comments of my own too with some ideas :wink:

andrewtavis commented on src/scribe_data/extract_transform/languages/English/formatted_data/translated_words.json:
```diff
@@ -0,0 +1,849 @@
+            "it": "Il riso",
+            "pt": "O arroz",
+            "ru": "Рис",
+            "es": "El arroz",
```
Happy to include it in here if @Linfye is comfortable with it :)

shashank-iitbhu replied:
> This could make sense, but maybe we could do a separate issue for it?

@andrewtavis I already have a branch in my local where i have moved the tranlations functions to `utils.py`, If you allow i can open a separate PR for this. or can open a PR for #77 russian translation process.
then other contributors can take reference from that.

andrewtavis replied:
A separate PR for this sounds good, @shashank-iitbhu! From there we can jump to the Russian :) Appreciate you reaching out here!

Linfye commented:
I fixed the problems you mentioned except the last one. @wkyoshida I wonder now should I work on it or others works on it now. Looking forward to your reply.
cc @andrewtavis

shashank-iitbhu commented:
> I fixed the problems you mentioned except the last one. @wkyoshida I wonder now should I work on it or others works on it now. Looking forward to your reply.
> cc @andrewtavis 

Can you refer to #88 and #89 ? I have implemented a different approach i.e batch processing of words for translation. This way it is relatively faster.

We can decide on a single approach, if the requirement is to iterate over each word rather than batch processing then we can go ahead with this PR.
cc @andrewtavis @wkyoshida

Linfye commented:
> > I fixed the problems you mentioned except the last one. @wkyoshida I wonder now should I work on it or others works on it now. Looking forward to your reply.
> > cc @andrewtavis
> 
> Can you refer to #88 and #89 ? I have implemented a different approach i.e batch processing of words for translation. This way it is relatively faster.
> 
> We can decide on a single approach, if the requirement is to iterate over each word rather than batch processing then we can go ahead with this PR. cc @andrewtavis @wkyoshida

I check the code and wonder if continue downloading from last progress cause the words are too much. If it works better, we can adopt yours.

andrewtavis commented:
Sorry for the delay on all of this, all :) I was on vacation and then sick right after... Checked and sent along some formatting in 2460584. I'll bring this in shortly as well as the work that's @shashank-iitbhu mentioned. I'll give it all a test to see how things are working. I'd say batch processing and having the process in the utils makes sense to me 😊

andrewtavis reviewed (approved):
Thanks for the great first contribution, @Linfye! Very important step in the the new translation process 😊

andrewtavis commented:
Ah, and a quick note on this: let's be sure to remove as much whitespace from JSON outputs as possible in the future as that does bring the file size down slightly 😊
